assume/markets/clearing_algorithms/__init__.py

```diff
@@ -1,13 +1,15 @@
 from assume.common.market_objects import market_mechanism
 
 from .all_or_nothing import pay_as_bid_aon, pay_as_clear_aon
-from .base import pay_as_bid, pay_as_clear
-from .pyomo_clearing import nodal_pricing_pyomo
+from .complex_clearing import pay_as_clear_opt
+from .nodal_pricing import nodal_pricing_example
+from .simple import pay_as_bid, pay_as_clear
```

Comment:
moving base to simple is fine
having a separate file per complex clearing does not make a lot of sense to me.
I liked they way of categorizing by the pyomo dependency better..?
Not a problem